src/search-backend/metadata-search-backend.ts

```diff
@@ -30,7 +30,7 @@ export class MetadataSearchBackend extends BaseSearchBackend {
       params
     );
     const queryAsString = urlSearchParam.toString();
-    const url = `https://${this.baseUrl}${this.servicePath}/?service_backend=metadata&${queryAsString}`;
```

Comment:
q: we no longer need to pass param?

Reply:
Right -- in fact, this param is the PPS default so it was never strictly required to begin with.

The new PPS calls for the profile page aren't currently allowed to include this `service_backend` param, so just omitting it seems reasonable.